### PR TITLE
Add psinghr Floral Park file

### LINKED ISSUE #57: Create a new file for psinghr Floral Park, NY
- [x] Create a branch
- [x] Add Commits
- [x] Open a PR
- [ ] Have a discussion
- [ ] Seek Approval
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by psinghr**
closes #57

### CODE REVIEW

File: psinghr-Floral_Park.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Floral Park, NY
+
+### Food Recommendations
```
Comment (lrayman) [inline psinghr-Floral_Park.md:3]: nothing to eat here?

**Comment by lrayman** [comment]
:helicopter: :baby:

File: psinghr-Floral_Park.md (hunk 1 above)
Comment (psinghr) [reply psinghr-Floral_Park.md:3]: no i'm starving here

**Review by lrayman — APPROVED** [review]
:100: looks good!

**Pull request merged**